content/get-started.md

```diff
@@ -2,6 +2,18 @@
 title: Get Started with Webpack
 ---
 
-* basic concepts, get webpack up and running, technical focus
-* target - the user should be able to complete this in 10 minutes
+`webpack` is a module bundler.A bundler is a tool that builds all your project files to a single asset file that can be loaded onto a browser.It requires that your application files be written as [modules](/concepts/modules).
```

Comment:
What does "builds all your project files" mean? We need a better word so beginners can get an idea what webpack can do.